packages/shared/styles/components/SfBullets.scss

```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$border-radius: 50%;
+$inactive-size: 8px;
+$active-size: 10px;
+$inactive-margin: 6px;
+$active-margin: 5px;
+$inactive-color: $c-gray-primary;
+$active-color: $c-green-secondary;
+
+.sf-bullets {
+    display: flex;
+}
+
+.sf-bullet {
+    border-radius: $border-radius;
+    background: $inactive-color;
+    width: $inactive-size;
+    height: $inactive-size;
+    margin: $inactive-margin;
+    cursor: pointer;
+    
+    &__active {
```

Comment:
`active` is a modifier for bullets element, not another child element, right?
following [BEM](http://getbem.com/introduction/) you should use `--` for modifiers, so i think `&--active` is the correct here.